server/rucio.conf.j2

```diff
@@ -52,6 +52,7 @@ CacheRoot /tmp
 {% else %}
  SSLCACertificateFile /etc/grid-security/ca.pem
 {% endif %}
+ SSLCARevocationCheck {{ RUCIO_CA_REVOCATION_CHECK | default('chain') }}
```

Comment:
Please put it below `SSLCARevocationPath {{ RUCIO_CA_PATH }}` in line 51. According to the documentation "SSLCARevocationPath" must be configured when this directive is set and if someone does not set "RUCIO_CA_PATH" then it might not work.